sdk/textanalytics/Azure.AI.TextAnalytics/src/AbstractSummaryAction.cs

```diff
@@ -52,9 +52,9 @@ public AbstractSummaryAction(AbstractSummaryOptions options)
         public string ActionName { get; set; }
 
         /// <summary>
-        /// The maximum number of sentences that the resulting summaries can have. If not set, the service default is
-        /// used.
+        /// The desired number of sentences in the resulting summaries, which the service will attempt to approximate.
+        /// If not set, the service default is used.
```

Comment:
Do we know what the service default is?

Reply:
Now that you mention it, we actually don't. I was under the impression that it was 3 like in the case of extractive summarization based on my test cases, but that might just be a coincidence since this value is just an approximation. I'll remove this part of the comment. In any case, the fact that it's approximate means that knowing about the default value is not as useful.